### ISSUE TITLE
Uploading syllabus and notice doesn’t store class number

### ISSUE THREAD

**Issue by Majidkadi** [changeweb/Unifiedtransform#223]
When I upload syllabus or notice, the class number shows 0 in class field of syllabuses and notices tables in database.

I’m using xampp on Windows 10 in development mode (local machine)

**Comment by changeweb** [comment]
This cached for 24hrs. Please read instructions about it in readme.md file

**Comment by Majidkadi** [comment]
Yes, but the problem is there is not class_id / course_id passed when storing (syllabus; routine; or notice).
I've checked the forms and in components.file.uploader, not parameters called (class_id...)

**Comment by changeweb** [comment]
`section_id` is passed with which `class_id` is used as foreign key. Here `section_id` is table column name, not Section number.

**Comment by Majidkadi** [comment]
I've just uploaded a syllabus for class 1 and it didn't show in the syllabuses list under that class after checking the database school_id => 1; user_id => 2 and class_id => 0. 
Same thing with routine and notice.
Procedure:
 From admin dashboard:
1. click on "Classes and Sections"
2. View syllabuses for this class
3. tape syllabus name in the field (syllabus upload test and choose) the file
4. Click on upload 
5. Message "Upload finished."
after refreshing it is not in the list and class_id is 0 in database syllabuses table.

**Comment by Majidkadi** [comment]
I'm trying to find out where is the problem, so in create function I took out the int word like this:
create($class_id) instead of create (int $class_id) and I get the id of thst class (my e because of the php vertion: I'm using the latest)
But in store function there is no $class_id passed in the list:
      $tb = new Syllabus;
      $tb->file_path = $request->file_path;
      $tb->title = $request->title;
      $tb->active = 1;
      $tb->school_id = \Auth::user()->school_id;
      $tb->user_id = \Auth::user()->id;
      //$tb->class_id = $class_id;/* *** */
      $tb->save();
Same thing in UploadController:
      $tb = new \App\Syllabus;
      $tb->file_path = 'storage/'.$path;
      $tb->title = $request->title;
      $tb->active = 1;
      $tb->school_id = auth()->user()->school_id;
      $tb->user_id = auth()->user()->id;
      $tb->save();

**Comment by changeweb** [comment]
I forgot to add these in Routine and Syllabus upload forms but I added the fields in Seeder files.

Now I have updated the code base with a commit. Update your code base with latest repository code.

**Comment by Majidkadi** [comment]
Thanks too much
